src/pages/Popup/Popup.jsx

```diff
@@ -72,8 +63,10 @@ const Popup = () => {
         </form>
       </div>
       <div className="Button-area">
-        <button className="Button-style Button-record" id="recordButton" type="button"><i className="fa-solid fa-circle-dot"></i> Record</button>
-        <button className="Button-style Button-play" id="playButton" type="button"><i className="fa-solid fa-circle-play"></i> Play</button>
+        {buttonRecordActive || recordButton}
```

Comment:
You can rewrite to something like:
```
{{ buttonRecordActive? recordButton: stopRecordButton}
{{ buttonPlayActive ? playButton : stopPlayButton}
```

Right?

Reply:
Oh I thought the other way is the way to do it in react :) 
You are right, let me update it.